packages/av/install

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -x
+set -e
+
+OS_DISTRIBUTION=$(lsb_release -cs)
+
+# need for add-apt-repository
+apt-get install -y software-properties-common
+add-apt-repository ppa:jonathonf/ffmpeg-3 -y
```

Comment:
How does that work for Bionic then, since `ffmpeg-3` doesn't support it?

Reply:
Bionic already has ffmpeg3 in it's main repos. You only need the PPA for Xenial and Trusty.

Reply:
That needs to be updated in https://github.com/rstudio/shinyapps-package-dependencies/blob/master/CONTRIBUTING.md since it says nothing about `bionic`.

Reply:
This is now added in the `if` statement for `xenial` and `trusty`